src/Shouldly/ShouldlyTimeoutException.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace Shouldly
+{
+    public class ShouldlyTimeoutException : Exception
```

Comment:
What effect (if any) would this have on backwards compatibility? (@JakeGinnivan may be able to answer this as he knows everything)